db/seeds.rb

```diff
@@ -7,13 +7,14 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
 if User.count.zero?
-  User.create(
+  User.create!(
     email: 'admin@example.com',
     password: 'passw0rd',
     password_confirmation: 'passw0rd',
     display_name: 'admin',
     biography: 'Administrator',
     admin: true,
+    username: 'admin-default'
```

Comment:
__[rubocop]__
* Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-params-comma)

<!-- Comment by [SideCI](https://sideci.com) -->